web/packages/build/vite/config.ts

```diff
@@ -140,13 +140,14 @@ export function createViteConfig(
           secure: false,
           ws: true,
         },
-        // /webapi/sites/:site/desktopplayback/:sid
-        '^(\\/v[0-9]+\\/webapi\\/sites\\/(.*?)\\/(desktopplayback|session-recording)\\/(.*?))(\\/ws)?':
+        // /webapi/sites/:site/(desktopplayback|sessionrecording|ttyplayback)/:sid
+        '^(\\/v[0-9]+\\/webapi\\/sites\\/(.*?)\\/(desktopplayback|sessionrecording|ttyplayback)\\/(.*?))(\\/ws)?':
           {
             target: `wss://${target}`,
             changeOrigin: true,
             secure: false,
             ws: true,
+            rewriteWsOrigin: true,
```

Comment:
nit: could you add a comment explaining what this does and why we need it here?

Reply:
done